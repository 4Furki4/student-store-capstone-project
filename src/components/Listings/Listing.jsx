/* eslint-disable @next/next/no-img-element */
import React from "react";

export default function Listing({ listing }) {
    return (
        <>
            <div
                className='align-center listingCard relative mx-auto w-[300px] 
            cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white
            pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue'
            >
                <div
                    className='absolute hidden h-full w-full bg-white p-6 text-center'
                    id='listingOverview'
                >
                    <h3 className='m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua'>
                        Listing Details
                    </h3>
                    <h3 className='text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue'>
                        Item Condition
                    </h3>
                    <p className='text-md m-2 inline-block text-hadrisblue'>
                        {listing.condition}
                    </p>
                    <h3 className='text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue'>
                        Listing Description
                    </h3>
                    <p className='text-md m-2 inline-block text-hadrisblue'>
                        {listing.description}
                    </p>
                </div>
                <div className='w-full overflow-hidden'>
                    <img src={listing.img} alt='image' className='object-fit' />
                </div>
                <div className='mt-4 flex justify-between px-3'>
                    <div className=' text-body-color text-md mb-2 leading-relaxed'>
                        {listing.title}
                    </div>
                    <span className='custom-bold text-lg'>
                        â‚º{listing.price}
                    </span>
                </div>
                <hr></hr>
                <div className='align-center mb-2 flex justify-between px-3'>
                    <h3>Listing Date:</h3>
                    <span className='custom-bold text-md text-center'>
                        {listing.listingDate}
                    </span>
                </div>
                <div className='mb-2 flex  justify-center p-2'>
                    <span className=' rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800'>
                        {listing.category}
                    </span>
                </div>
            </div>
        </>
    );
}
