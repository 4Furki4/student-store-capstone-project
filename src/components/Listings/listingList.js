const listingList = [
    {
        id: 1,
        img: "https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg",
        title: "Apple iPad Pro ",
        price: "4000",
        category: "ELECTRONICS",
        location: "Istanbul",
        listingDate: "2022-04-01",
        description:
            "Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches",
        condition: "Used-Good Condition",
    },
    {
        id: 2,
        img: "https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg",
        title: "Leather Couch",
        price: "1000",
        category: "FURNITURE",
        location: "Izmir",
        listingDate: "2022-07-01",
        description: "Leather Couch - Brown, doesn't have stains",
        condition: "Used-Mint Condition",
    },
    {
        id: 3,
        img: "https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg",
        title: "Apple iPad Pro ",
        price: "4000",
        category: "ELECTRONICS",
        location: "Istanbul",
        listingDate: "2022-04-01",
        description:
            "Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches",
        condition: "Used-Good Condition",
    },
    {
        id: 4,
        img: "https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg",
        title: "Leather Couch",
        price: "1000",
        category: "FURNITURE",
        location: "Izmir",
        listingDate: "2022-07-01",
        description: "Leather Couch - Brown, doesn't have stains",
        condition: "Used-Mint Condition",
    },
    {
        id: 5,
        img: "https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg",
        title: "Apple iPad Pro ",
        price: "4000",
        category: "ELECTRONICS",
        location: "Istanbul",
        listingDate: "2022-04-01",
        description:
            "Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches",
        condition: "Used-Good Condition",
    },
    {
        id: 6,
        img: "https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg",
        title: "Leather Couch",
        price: "1000",
        category: "FURNITURE",
        location: "Izmir",
        listingDate: "2022-07-01",
        description: "Leather Couch - Brown, doesn't have stains",
        condition: "Used-Mint Condition",
    },
    {
        id: 7,
        img: "https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg",
        title: "Apple iPad Pro ",
        price: "4000",
        category: "ELECTRONICS",
        location: "Istanbul",
        listingDate: "2022-04-01",
        description:
            "Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches",
        condition: "Used-Good Condition",
    },
    {
        id: 8,
        img: "https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg",
        title: "Leather Couch",
        price: "1000",
        category: "FURNITURE",
        location: "Izmir",
        listingDate: "2022-07-01",
        description: "Leather Couch - Brown, doesn't have stains",
        condition: "Used-Mint Condition",
    },
    {
        id: 9,
        img: "https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg",
        title: "Apple iPad Pro ",
        price: "4000",
        category: "ELECTRONICS",
        location: "Istanbul",
        listingDate: "2022-04-01",
        description:
            "Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches",
        condition: "Used-Good Condition",
    },
    {
        id: 10,
        img: "https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg",
        title: "Leather Couch",
        price: "1000",
        category: "FURNITURE",
        location: "Izmir",
        listingDate: "2022-07-01",
        description: "Leather Couch - Brown, doesn't have stains",
        condition: "Used-Mint Condition",
    },
];

export default listingList;
