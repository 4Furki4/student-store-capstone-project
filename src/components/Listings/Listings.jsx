import React from "react";

import Listing from "./Listing";
import listingList from "./listingList";

export default function Listings() {
    return (
        <div className='align-center flex w-full flex-wrap justify-start gap-6 bg-iceblue p-10 align-top'>
            {listingList.map((listing) => {
                return <Listing listing={listing} key={listing.id} />;
            })}
        </div>
    );
}
