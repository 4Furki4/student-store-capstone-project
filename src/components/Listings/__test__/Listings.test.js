import renderer from "react-test-renderer";

import Listings from "../Listings";

it("renders correctly", () => {
    const tree = renderer.create(<Listings />).toJSON();
    expect(tree).toMatchSnapshot();
});
