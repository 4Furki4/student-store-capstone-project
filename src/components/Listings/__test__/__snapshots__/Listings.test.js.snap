// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="align-center flex h-screen w-full flex-wrap justify-start gap-6 overflow-scroll bg-iceblue p-10 align-top"
>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Good Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Apple iPad Pro 
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        4000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-04-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        ELECTRONICS
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Mint Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Leather Couch - Brown, doesn't have stains
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Leather Couch
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        1000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-07-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        FURNITURE
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Good Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Apple iPad Pro 
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        4000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-04-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        ELECTRONICS
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Mint Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Leather Couch - Brown, doesn't have stains
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Leather Couch
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        1000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-07-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        FURNITURE
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Good Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Apple iPad Pro 
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        4000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-04-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        ELECTRONICS
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Mint Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Leather Couch - Brown, doesn't have stains
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Leather Couch
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        1000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-07-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        FURNITURE
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Good Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Apple iPad Pro 
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        4000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-04-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        ELECTRONICS
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Mint Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Leather Couch - Brown, doesn't have stains
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Leather Couch
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        1000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-07-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        FURNITURE
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Good Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Apple iPad Pro - 12.9-inch - Wi-Fi - 64GB - Space Gray, minor scratches
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://productimages.hepsiburada.net/s/32/375/10354042765362.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Apple iPad Pro 
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        4000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-04-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        ELECTRONICS
      </span>
    </div>
  </div>
  <div
    className="align-center listingCard relative mx-auto w-[300px] cursor-pointer flex-col justify-center overflow-hidden rounded-lg bg-white pb-2 font-Poppins shadow-xl transition duration-500 hover:scale-105 hover:shadow-hadrisblue"
  >
    <div
      className="absolute hidden h-full w-full bg-white p-6 text-center"
      id="listingOverview"
    >
      <h3
        className="m-2 rounded-xl bg-hadrisblue text-lg font-semibold text-cyanaqua"
      >
        Listing Details
      </h3>
      <h3
        className="text-md m-2 mt-6 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Item Condition
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Used-Mint Condition
      </p>
      <h3
        className="text-md m-2 rounded-xl bg-cyanaqua text-hadrisblue"
      >
        Listing Description
      </h3>
      <p
        className="text-md m-2 inline-block text-hadrisblue"
      >
        Leather Couch - Brown, doesn't have stains
      </p>
    </div>
    <div
      className="w-full overflow-hidden"
    >
      <img
        alt="image"
        className="object-fit"
        src="https://pyxis.nymag.com/v1/imgs/86c/882/b47a32d8e72df7aeadf74490a82aaa9432.rsquare.w600.jpg"
      />
    </div>
    <div
      className="mt-4 flex justify-between px-3"
    >
      <div
        className=" text-body-color text-md mb-2 leading-relaxed"
      >
        Leather Couch
      </div>
      <span
        className="custom-bold text-lg"
      >
        ₺
        1000
      </span>
    </div>
    <hr />
    <div
      className="align-center mb-2 flex justify-between px-3"
    >
      <h3>
        Listing Date:
      </h3>
      <span
        className="custom-bold text-md text-center"
      >
        2022-07-01
      </span>
    </div>
    <div
      className="mb-2 flex justify-center p-2"
    >
      <span
        className=" rounded-full bg-teal-200 px-3 text-sm font-semibold uppercase tracking-wide text-teal-800"
      >
        FURNITURE
      </span>
    </div>
  </div>
</div>
`;
